tests/helpers/destroy-app.js

```diff
@@ -1,6 +1,4 @@
-import Ember from 'ember';
```

Comment:
If you decide we should update the test suite as a part of this PR, no Ember blueprint helpers need to be left in the `tests` dir.